clang/test/OpenMP/target_map_array_of_structs_with_nested_mapper_ast_dump.cpp

```diff
@@ -0,0 +1,34 @@
+//RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fopenmp -ast-dump  %s | FileCheck %s --check-prefix=DUM
+
+typedef struct {
+  int a;
+} C;
+#pragma omp declare mapper(C s) map(to : s.a)
+
+typedef struct {
+  int e;
+  C f;
+  int h;
+} D;
+
+void foo() {
+  D sa[10];
+  sa[1].e = 111;
+  sa[1].f.a = 222;
+
+#pragma omp target map(tofrom : sa)
```

Comment:
The test is identical to the array-section version, with the only difference being this line: https://github.com/llvm/llvm-project/blob/6f64a600649a95141526043b170aa4244d54c94b/clang/test/OpenMP/target_map_nest_defalut_mapper_ast_dump.cpp#L19